plugins/render/api_key.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+type Config struct {
+	Version           int
+	SSHPreserveHosts  bool `yaml:"sshPreserveHosts"`
+	Profiles          map[string]Profile
+}
+
+type Profile struct {
+	DefaultRegion string `yaml:"defaultRegion"`
+	APIKey        string `yaml:"apiKey"`
+}
```

Comment:
Looks like nor the importer either the provisioner use `defaultRegion`, is there a reason to keep it around?

Reply:
updated